hex/helpers/lib/do_update.exs

```diff
@@ -31,6 +31,22 @@ dependency = String.to_atom(dependency_name)
 {dependency_lock, rest_lock} = Map.split(Mix.Dep.Lock.read(), [dependency])
 Mix.Dep.Fetcher.by_name([dependency_name], dependency_lock, rest_lock, [])
 
+System.cmd(
+  "mix",
+  [
+    "deps.get",
+    "--no-compile",
+    "--no-elixir-version-check",
```

Comment:
Xref https://github.com/dependabot/dependabot-core/blob/main/hex/helpers/lib/run.exs#L54-L57 for the source.

I didn't think `--no-deps-check` was relevant as the intent is to... check the deps.

I thought `--no-start` may only be relevant to `mix run...`?

Reply:
This makes sense to me 👍